scripts/fast-reboot

```diff
@@ -41,6 +41,15 @@ docker kill lldp > /dev/null
 # Kill teamd, otherwise it gets down all LAGs
 docker kill teamd > /dev/null
 
+# Gracefully stop syncd
+docker exec -ti syncd /usr/bin/syncd_request_shutdown --cold > /dev/null
+
+# Check it was stopped
+until [[ $(docker top syncd | grep -q /usr/bin/syncd) -eq 0 ]]
```

Comment:
I think you need to s/until/while/

Reply:
Both `grep -q` and `grep -c` will return 0 if any matches are found, and will return 1 if no matches are found.

Therefore, either are applicable, however, the condition needs to be changed to either

`until [[ $(docker top syncd | grep -q /usr/bin/syncd) -eq 1 ]]`

or 

`while [[ $(docker top syncd | grep -q /usr/bin/syncd) -eq 0 ]]`

Reply:
$(...) is getting the output of the execution. Therefore, with -q it is always ''.

With -c, the output would be how many lines find. We want to wait until there is 0 lines.

Reply:
You're right, Ying. For some reason I was stuck on the return value, not the stdout output.